Cargo.toml

```diff
@@ -19,4 +19,19 @@ rust-version = "1.71.1"
 thiserror = "2"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-attohttpc = { version = "0.28", features = ["json"] }
+attohttpc = { version = "0.28", default-features = false, features = ["json", "compress"] }
+
+[features]
+default = ["tls-rustls"]
+
+# Set the TLS to native-tls (Bumps MSRV to <= 1.80.0)
```

Comment:
Shouldn't this say `>= 1.80.0`?

Reply:
Same for the other comment.

Reply:
The worse part is that I wrote it correctly, then though it was the other way around. Big derp

Reply:
Fixed and rebased